mrscake/src/components/BakeryRegistration.jsx

```diff
@@ -1,74 +0,0 @@
-import React, { Component, Input } from 'react';
```

Comment:
@Parul76 why we delete this? @niloufar-rashedi was it your task, right? Do we need this code or delete?

Reply:
import React, { Component, Input } from 'react'; should not be deleted; it is related to the SignInSignUp form in bakery registration form